### PR TITLE
Update HTTP, Net, Process API docs

### PR DESCRIPTION
**Pull request by daeyeon**
IoT.js-DCO-1.0-Signed-off-by: Daeyeon Jeong daeyeon.jeong@samsung.com

### CODE REVIEW

File: docs/api/IoT.js-API-HTTP.md [hunk 1]
```diff
@@ -1,62 +1,122 @@
-# Module: http
+### Platform Support
+
+
+<a id="markdown-contents" name="contents"></a>
+### Contents
+
+- [http](#http)
```
Comment (hs0225) [inline docs/api/IoT.js-API-HTTP.md]: Please change to upper case.

File: docs/api/IoT.js-API-HTTP.md [hunk 2]
```diff
@@ -1,62 +1,122 @@
...
+        - [`statusMessage`](#statusmessage)
+        - [`socket`](#socket)
+    - [Prototype Functions](#prototype-functions-3)
+        - [`setTimeout(ms, cb)`](#settimeoutms-cb-3)
+    - [Events](#events-3)
+        - [`'end'`](#end-1)
+        - [`'close'`](#close-2)
+
+<a id="markdown-http" name="http"></a>
+# http
```
Comment (hs0225) [inline docs/api/IoT.js-API-HTTP.md]: ditto

File: docs/api/IoT.js-API-Net.md [hunk 3]
```diff
@@ -23,234 +22,305 @@ The following shows net module APIs available for each platform.
...
-#### net.connect(port[, host][, connectListener])
-#### net.createConnection(options[, connectListener])
-#### net.createConnection(port[, host][, connectListener])
-* `options: Object`
-* `port: Number`
-* `host: String`, Default: `localhost`
-* `connectListener: Function()`
+### Contents
+
+- [net](#net)
```
Comment (hs0225) [inline docs/api/IoT.js-API-Net.md]: Please change the first letter to upper case.

File: docs/api/IoT.js-API-Net.md [hunk 4]
```diff
@@ -23,234 +22,305 @@ The following shows net module APIs available for each platform.
...
 Creates a `net.Socket` and connects to the supplied host.
 
 It is equivalent to `new net.Socket()` followed by `socket.connect()`.
 
-
-#### net.createServer([options][, connectionListener])
-* `options: Object`
-* `connectionListener: Function(connection: net.Socket)`
+<a id="markdown-netcreateserveroptions-connectionlistener" name="netcreateserveroptions-connectionlistener"></a>
+### net.createServer([options][, connectionListener])
```
Comment (hs0225) [inline docs/api/IoT.js-API-Net.md]: Please wrap `

File: docs/api/IoT.js-API-Net.md [hunk 5]
```diff
@@ -23,234 +22,305 @@ The following shows net module APIs available for each platform.
...
 
-### Constructor
+This object is an abstraction of a TCP or local socket. net.Socket inherits [`Stream.Duplex`](IoT.js-API-Stream.md). They can be created by the user and used as a client (with connect()) or they can be created by IoT.js and passed to the user through the 'connection' event of a server.
 
-#### new net.Socket([options])
-* `options: Object`
+<a id="markdown-constructor" name="constructor"></a>
+## Constructor
+<a id="markdown-new-netsocketoptions" name="new-netsocketoptions"></a>
+### new net.Socket([options])
```
Comment (hs0225) [inline docs/api/IoT.js-API-Net.md]: ditto.

File: docs/api/IoT.js-API-Net.md [hunk 6]
```diff
@@ -23,234 +22,305 @@ The following shows net module APIs available for each platform.
...
 
-If the socket is inactive for `timeout` milliseconds, `'timeout'` event will emit.
 
-`callback` is registered as `timeout` event listener.
+<a id="markdown-lookup" name="lookup"></a>
+### `'lookup'`
+* `callback <Function(err, address, family)>`
+* `err <Error> | Null`
+* `address <String>`
+* `family <String> | Null`
```
Comment (hs0225) [inline docs/api/IoT.js-API-Net.md]: Please insert two spaces.

File: docs/api/IoT.js-API-Net.md [hunk 7]
```diff
@@ -23,234 +22,305 @@ The following shows net module APIs available for each platform.
...
+        - [`'close'`](#close-1)
+        - [`'data'`](#data)
+        - [`'drain'`](#drain)
+        - [`'end'`](#end)
+        - [`'error'`](#error-1)
+        - [`'lookup'`](#lookup)
+        - [`'timeout'`](#timeout)
+
+<a id="markdown-net" name="net"></a>
+# net
```
Comment (hs0225) [inline docs/api/IoT.js-API-Net.md]: ditto.

File: docs/api/IoT.js-API-HTTP.md (hunk 1 above)
Comment (daeyeon) [inline docs/api/IoT.js-API-HTTP.md]: done

File: docs/api/IoT.js-API-HTTP.md (hunk 2 above)
Comment (daeyeon) [inline docs/api/IoT.js-API-HTTP.md]: done

File: docs/api/IoT.js-API-Net.md (hunk 3 above)
Comment (daeyeon) [inline docs/api/IoT.js-API-Net.md]: done

File: docs/api/IoT.js-API-Net.md (hunk 7 above)
Comment (daeyeon) [inline docs/api/IoT.js-API-Net.md]: done

File: docs/api/IoT.js-API-Net.md (hunk 6 above)
Comment (daeyeon) [inline docs/api/IoT.js-API-Net.md]: Every review was changed except this.

**Comment by daeyeon** [comment]
Every wrong case has been fixed as reviewed.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**